test/test_cli.py

```diff
@@ -32,10 +32,16 @@ def test_makedirs_overwrite_soft():
     assert _stat.st_ino == stat.st_ino
 
 
-@pytest.mark.skipif(sys.platform == 'win32',
-                    reason='st_ino does not work on Windows')
-def test_makedirs_overwrite_hard():
-    _directory, _stat = mkdtemp().next()
-    makedirs(_directory, overwrite='hard')
-    stat = os.stat(_directory)
-    assert _stat.st_ino != stat.st_ino
+
+def test_cli_overwrite_hard():
+    _directory, _ = mkdtemp().next()
+    args = {
+        'INDIR': _directory,
+        '--mode': 'production',
+        '--overwrite': 'hard'
+    }
+    cli(args)
+    assert os.path.isdir(os.path.join(_directory, 'production_configs'))
+    args.update({'--mode': 'development'})
+    cli(args)
+    assert not os.path.isdir(os.path.join(_directory, 'production_configs'))
```

Comment:
This sequence of `assert`s check if an instance created under `production` mode lost the `production_configs` when it was overwritten by a second call with `development`. This is not an ideal test b/c it test a corner case only, but it works on Windows and we can add additional cases later. (For example, a test that modifies the contents of the config files and see if the expected modification are there, or not, when calling the `cli`).